src/components/Button.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const styles = {
+  right: {
+    backgroundColor: "orange",
+    color: "white"
+  },
+  left: {
+    backgroundColor: "gainsboro",
+    color: "black"
+  }
+};
+
+const sizes = ["58.2px", "116.6px"];
+
+const ButtonLayout = styled.button`
+  width: ${props => props.cellSize};
+  height: 47.5px;
+  background-color: ${props => props.backgroundColor};
+  font-size: 22px;
+  color: ${props => props.color};
+`;
+
+const Button = props => (
+  <ButtonLayout
+    backgroundColor={styles[props.position]["backgroundColor"]}
+    cellSize={sizes[Number(props.cellSize) - 1 || 0]}
```

Comment:
Please use `size` instead of `cellSize`. Also please use `small` and `large` as possible values for `size`  and make the one of them the default value. 0 and 1 don't say much about what the size of the button will be.